.bootstrapper/SemVerBootstrapper.ps1

```diff
@@ -0,0 +1,8 @@
+#requires -version 3.0
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+if (Get-Command dotnet -ErrorAction SilentlyContinue) {
+    dotnet publish -o .
```

Comment:
doesn't this mean every user needs to have the dotnet SDK/CLI installed?

Reply:
Yes, but the intent would be that it is only run on the build server or by someone who's cloned it.

Reply:
Oh, got it. I find this kinda weird to be honest. I've never seen a software that had code in the source to check whether the dependencies are installed and if not does that. I would rather expect the readme or contribution guide to show the command to run to install dependencies, next to the command how to run tests. `Install-Module Pester` is another thing that one has to do to run the tests anyway.